packages/elements/src/account/elements/logto-account-center.ts

```diff
@@ -0,0 +1,56 @@
+import { consume } from '@lit/context';
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+
+import {
+  logtoAccountContext,
+  type LogtoAccountContextType,
+} from '../providers/logto-account-provider.js';
+
+const tagName = 'logto-account-center';
+
+@customElement(tagName)
+export class LogtoAccountCenter extends LitElement {
+  static tagName = tagName;
```

Comment:
nit: since this is a static const property already, we can directly set the string value here.

Reply:
The tagName comes from a single place because it’s used in multiple locations, making it easier to maintain.